### PR TITLE
added 2 questions in html

### PR DESCRIPTION
**Pull request by achuehuehue**
## PR Checklist

This PR is ready for review and meets the requirements set out
in [Suggestion how to contribute](CONTRIBUTING.md)

### DOD

- [x] I have added new quiz{'s}
- [ ] I have added new reference link{'s}
- [ ] I have made small correction/improvements

### Changes / Instructions

_Add instructions to me, please type here, thanks_

### CODE REVIEW

File: html/html-quiz.md [hunk 1]
```diff
@@ -2219,3 +2219,33 @@ label="Endnotes">Endnotes...</custom-footnote>
...
+
+### Q136.Which of the following HTML element is used for creating an unordered list?
+
+- [ ] <ui>
+
+- [ ] <i>
+
+- [ ] <em>
+
+- [ ] <ul>
```
Comment (haxkd) [inline html/html-quiz.md:2244]: mark the answer too

**Review by haxkd — CHANGES_REQUESTED** [review]

**Comment by haxkd** [comment]
@achuehuehue resolve merge conflict

File: html/html-quiz.md (hunk 1 above)
Comment (achuehuehue) [reply html/html-quiz.md]: marked

Comment (haxkd) [reply html/html-quiz.md]: Resolve merge conflict

**Review by Ebazhanov — APPROVED** [review]
LGTM, thank you @achuehuehue 👍

**Pull request merged**